web-prover-core/src/parser/config.rs

```diff
@@ -100,27 +127,27 @@ mod tests {
             "name": "John Doe"
             // Missing age and active fields
         }
-    });
+    })
+    .to_string()
+    .into_bytes();
     let config = ExtractorConfig {
       format:     DataFormat::Json,
       extractors: vec![
         name_extractor(),
-        Extractor {
-          id:             "userAge".to_string(),
-          description:    "Extract user's age".to_string(),
-          selector:       vec!["user".to_string(), "age".to_string()],
-          extractor_type: ExtractorType::Number,
-          required:       false, // Optional
-          predicates:     vec![],
-        },
-        Extractor {
-          id:             "userActive".to_string(),
-          description:    "Extract user's active status".to_string(),
-          selector:       vec!["user".to_string(), "active".to_string()],
-          extractor_type: ExtractorType::Boolean,
-          required:       false, // Optional
-          predicates:     vec![],
-        },
+        extractor!(
```

Comment:
why do we want to use a macro here?

Reply:
Macros feel like a last resort to me, but i bet you have a good reason

Reply:
No particular reason. I use macros as a constructor in tests. IIRC this was for consistency.